src/modules/badWords.json

```diff
@@ -15,12 +16,23 @@
     "jebany": "uprawiający sex",
     "jebana": "uprawiająca sex",
 
+    "sex":"W trosce o byt i przyszłość naszej Ojczyzny, odzyskawszy w 1989 roku możliwość suwerennego i demokratycznego stanowienia o Jej losie, my, Naród Polski - wszyscy obywatele Rzeczypospolitej, zarówno wierzący w Boga będącego źródłem prawdy, sprawiedliwości, dobra i piękna, jak i nie podzielający tej wiary, a te uniwersalne wartości wywodzący z innych źródeł, równi w prawach i w powinnościach wobec dobra wspólnego - Polski, wdzięczni naszym przodkom za ich pracę, za walkę o niepodległość okupioną ogromnymi ofiarami, za kulturę zakorzenioną w chrześcijańskim dziedzictwie Narodu i ogólnoludzkich wartościach, nawiązując do najlepszych tradycji Pierwszej i Drugiej Rzeczypospolitej, zobowiązani, by przekazać przyszłym pokoleniom wszystko, co cenne z ponad tysiącletniego dorobku, złączeni więzami wspólnoty z naszymi rodakami rozsianymi po świecie, świadomi potrzeby współpracy ze wszystkimi krajami dla dobra Rodziny Ludzkiej, pomni gorzkich doświadczeń z czasów, gdy podstawowe wolności i prawa człowieka były w naszej Ojczyźnie łamane, pragnąc na zawsze zagwarantować prawa obywatelskie, a działaniu instytucji publicznych zapewnić rzetelność i sprawność, w poczuciu odpowiedzialności przed Bogiem lub przed własnym sumieniem, ustanawiamy Konstytucję Rzeczypospolitej Polskiej jako prawa podstawowe dla państwa oparte na poszanowaniu wolności i sprawiedliwości, współdziałaniu władz, dialogu społecznym oraz na zasadzie pomocniczości umacniającej uprawnienia obywateli i ich wspólnot. Wszystkich, którzy dla dobra Trzeciej Rzeczypospolitej tę Konstytucję będą stosowali, wzywamy, aby czynili to, dbając o zachowanie przyrodzonej godności człowieka, jego prawa do wolności i obowiązku solidarności z innymi, a poszanowanie tych zasad mieli za niewzruszoną podstawę Rzeczypospolitej Polskiej.",
```

Comment:
Zdecydowanie zbyt długie, uniemozliwia to zacenzurowania wiadomosci w ktorej pojawia się te słowo 2 razy
Ja bym po prostu przetłumaczył "sex" jako płeć